docs/governance/community-guidelines/releases/cmb.md

```diff
@@ -0,0 +1,106 @@
+  - Relay the CR to all participating reviewers
+  - At least two members must agree on a CR before it moves forward, with majority opinion ruling when there is dissent.
+  - When discussion has been stabilized for at least 48 hours, determine the status of the CR
+  - If the CR is affirmed by the CMB, create a GitHub issue detailing the CR. Tag and notify the appropriate working group.
+  - If the CR is not affirmed by the CMB, notify the change requestor. The CR should not be resubmitted unless there are substantial changes to the request.
+- When all outstanding requests have been resolved and requested changes have been applied, initiate the release.
+  - Ensure that the release is no sooner than the expected delivery date, and that all actions follow the current processes of the [Delivery WG].
+
+### Reviewer Responsibilities
+
+Members are **not** obligated to review every release but will be notified and may choose to engage in reviews.
+
+When engaging, the following is expected of a CMB member:
+
+- Be thorough, thoughtful, and provide detailed feedback before requesting changes.
+  - Gather feedback from colleagues as needed to support a review.
+- If changes are requested, communicate clearly and promptly through the channels outlined by the Release Manager for the current release cycle.
+  - When a change request (CR) is received, the Release Manager will open discussions and facilitate responses from the board.
+- Members are encouraged to respond within 7 days if they have input on a CR.
+  - The Release Manager logs any dissenting opinions and communicates the majority decision.
+- A release cannot proceed without 5 approvals; members are encouraged to help meet this threshold by approving or requesting changes.
+
+### Qualifications for Participation
+
+Individuals of any background or experience level may participate in a review.
+
+To approve or request changes, an individual must be an appointed CMB member in good standing.
+
+CMB members are appointed by the [Delivery WG]. If you are interested or have any questions, please reach out to a current [Delivery WG] member or join the community call.
+
+### Release Manager Qualifications
+
+A release manager shall be a [Delivery WG] approver or a CMB member who has provided feedback on a previous release cycle.
+
+Release managers are expected to demonstrate the following qualities:
+
+- Strong written communication skills
+- High attention to detail
+- Commitment to process and protocol
+- Ability to parse and relay complex feedback
+- Fundamental knowledge of the domain featured in the release
+- Reasonable availability and responsiveness during the release cycle (at least one month)
+
+### Breach of Decorum
+
+Members of the Change Management Board are expected to follow the [FINOS Community Code of Conduct](https://community.finos.org/docs/governance/code-of-conduct) at all times.
+
+Appointments shall be permanently revoked in the following cases:
```

Comment:
Should we add anything about a lack of engagement here? i.e. revoke their status if they don't engage at all. Or is this not needed?

Reply:
Shouldn't be necessary, as there is no community impact to low engagement from CMB members and they have no individual responsibilities